lexis/render/clipPlanes.fbs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+
+// This event is used to communicate clip planes between
+// applications. Every plane is defined in point normal
+// form. n normal and d value [ nx, ny, nz d ]. Here for
+// given points, p, n.p + d = 0 is satisfied.
+
+// The normal is defined in world space and applications
+// should understand the reference frame to have meaningful
+// plane equations. The d value should be re-scaled by the
+// scaling factor of the application using the event.
```

Comment:
I would simply say that the plane equation is in world reference system, which is the same as the lookout event.

The rest are details that anybody doing graphics should know (if you don't you shouldn't be writing code using it).